app/facades/location_facade.rb

```diff
@@ -0,0 +1,12 @@
+class LocationFacade
+  def self.get_location(location, station)
+    station_address = station.map do |s|
+      s.address
+    end
+    to_address = station_address.join(', ')
+    location = LocationService.call_for_location(location, to_address)
+    location.map do |data|
+      Direction.new(data.second)
+    end
```

Comment:
I would break this method up into at least 3 separate methods to keep it more SRP.